Prospress-WC/ruleset.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<ruleset name="Prospress-WC">
+	<description>Prospress WooCommerce Coding Standards</description>
+
+	<!-- Include the main Prospress stnadard -->
+	<rule ref="Prospress"/>
+
+	<!-- Modify the behavior of the checks for escaping functions -->
+	<rule ref="WordPress.Security.EscapeOutput">
+		<!-- ignore wcs plain text email templates - strip_tags is run over the output amongst other things -->
+		<exclude-pattern>*/templates/emails/plain/*</exclude-pattern>
+		<properties>
+			<!-- e.g. body_class, the_content, the_excerpt -->
+			<property name="customAutoEscapedFunctions" type="array" value="woocommerce_wp_select,wcs_help_tip"/>
```

Comment:
💡  This [release note](https://github.com/squizlabs/PHP_CodeSniffer/releases/tag/3.3.0)  says that the above form of specifying array values has been deprecated and `element` tags should be used. Not sure if it is only in the case of key-value pairs.

Reply:
Thanks for finding that, I didn't know that was the case 🙂